### PR TITLE
Add Xubuntu to distribution

### PR DESCRIPTION
**Pull request by azeupaul**
I have added the xubuntu distribution to project

### CODE REVIEW

File: data/distribs.json [hunk 1]
```diff
@@ -52,5 +52,11 @@
         "main_color": "dd4814",
         "website": "http://www.ubuntu.com",
         "doc": "https://doc.ubuntu-fr.org"
+    },
+    "xubuntu": {
+        "name": "Xubuntu",
+        "main_color": "333333",
```
Comment (Huluti) [inline data/distribs.json]: For me, the main color of the distribution is a dark blue (see https://xubuntu.org and its logo).

**Comment by Huluti** [comment]
Hi! Thank's for your interest. Just change the main color and I merge the PR.

File: data/distribs.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
     },
     "xubuntu": {
         "name": "Xubuntu",
-        "main_color": "333333",
+        "main_color": "0044AA",
```
Comment (azeupaul) [inline data/distribs.json:58]: I have change

File: data/distribs.json (hunk 1 above)
Comment (azeupaul) [reply data/distribs.json]: It's done yet...

**Comment by Huluti** [comment]
I finally made the choice to return to the black. You were right !

**Comment by azeupaul** [comment]
Indeed, it's  a recommended main color on their [website](http://xubuntu.org/resources#logo_wordmark) with this logo

**Pull request merged**